app/src/main/java/com/example/marvelapp/fragment/CharacterFragment.kt

```diff
@@ -41,7 +40,7 @@ class CharacterFragment : DialogFragment(), KoinComponent {
         }
     }
 
-    private fun displayFragment(id: Character?) {
+    private fun displayFragment(id: com.example.domain.entity.Character?) {
```

Comment:
add the correct import instead of writing the path here

Reply:
done